plugins/mcumgr/smp_processor.cpp

```diff
@@ -191,6 +182,31 @@ void smp_processor::message_timeout()
     transport->send(last_message);
 }
 
+int smp_processor::get_group_handler_idx(uint16_t group){
+    log_debug() << "finding handler, group is" << group << " last sender " << last_sender_group_id;
+    if(last_sender_group_id != (uint32_t)group) 
+    {
+        return -1;
+    }
+    int i = 0;
```

Comment:
nit. newlines, and generally variable declaration at beginning of function (some existing code needs this fixed)